how-to/workspace-platform-starter/client/src/framework/platform/platform-mapper.ts

```diff
@@ -44,7 +50,8 @@ type WorkspacePlatformKeys =
 	| "preventPageDragOut"
 	| "preventPageDrag"
 	| "preventPageClose"
-	| "_internalDeferShowOptions";
+	| "_internalDeferShowOptions"
```

Comment:
should we exclude internal ones? We don't know what runtime or workspace do with those i.e. there may be scenarios where it doesn't work with the defaults.

Reply:
Only removed if its not the default value